### DISCUSSION TITLE
SUPORT

### DISCUSSION THREAD

**Discussion by pandasoli** [jellydn/fresh-deno-app#3] [General]
I got too much questions about Fresh, I think it would be nice a Discord server for every Fresh users talk about and maybe you talk too and explain somethings about Fresh.

Please, Fresh is limited about for styling, I need to know what's the options besides TWind.

**Comment by jellydn** [comment]
> I got too much questions about Fresh, I think it would be nice a Discord server for every Fresh users talk about and maybe you talk too and explain somethings about Fresh.

I opened the discussion on this repo for everyone wants to discuss about Fresh.

> Please, Fresh is limited about for styling, I need to know what's the options besides TWind.

Fresh is pretty new at this moment and if you look into `CSS` on https://github.com/denoland/fresh/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+style you will see a lot of open issues with this one. Last time, I see a WIP topic about styling when Fresh was in beta.
